base/basePull.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash -e
+readonly CPP_IMAGE_NAME="drydock/u14cppall"
```

Comment:
this is what is wrong. We should not touch basePull. Its only execPull that we should change for now